demonstrations/tutorial_neutral_atoms.py

```diff
@@ -0,0 +1,684 @@
+r""".. _neutral:
+
+Neutral-atom quantum computers
+=============================
+
+.. meta::
+    :property="og:description": Learn how neutral atom quantum devices work using code
+    :property="og:image": https://pennylane.ai/qml/_images/thumbnail_neutral_atoms.png
+
+.. related::
+   tutorial_pasqal Quantum computation with neutral atoms
+   tutorial_trapped_ions Trapped ion quantum computing
+   tutorial_sc_qubits Quantum computing with superconducting qubits
+   tutorial_photonics Photonic quantum computing
+
+*Author: Alvaro Ballon — Posted: 30 May 2023.*
+
+In the last few years, a new quantum technology has gained the attention of the quantum computing
+community. Thanks to recent developments in optical-tweezer technology,
+neutral atoms can be used as robust and versatile qubits. In 2020, a collaboration between
+various academic institutions produced a neutral-atom device with a whooping 256 qubits 😲! It is 
```

Comment:
```suggestion
various academic institutions produced a neutral-atom device with a whopping 256 qubits 😲! It is 
```
whooping is a sound, whopping is a lot 😄